js/templates/modals/wallet/coinNavItem.html

```diff
@@ -8,17 +8,19 @@
     if (e instanceof ob.NoExchangeRateDataError) {
      // pass - we'll just show the unconverted amount if the exchange rate data isn't
      // available
-    } else {
-      throw e; // let others bubble up
     }
   }
 
-  const formattedBalance = convertedCurrency === undefined ?
-    ob.currencyMod.formatCurrency(ob.balance, mnCode, { maxDisplayDecimals: 4 }) :
-    ob.currencyMod.formatCurrency(convertedCurrency, ob.displayCur,
-      { maxDisplayDecimals: ob.currencyMod.isFiatCur(ob.displayCur) ? 2 : 4 });
+  let formattedBalance = '';
 
-  const balanceText = ob.balance ?
+  if (typeof ob.balance === 'number') {
+    formattedBalance = convertedCurrency === undefined ?
+      ob.currencyMod.formatCurrency(ob.balance, mnCode, { maxDisplayDecimals: 4 }) :
+      ob.currencyMod.formatCurrency(convertedCurrency, ob.displayCur,
+        { maxDisplayDecimals: ob.currencyMod.isFiatCur(ob.displayCur) ? 2 : 4 });
+  }
+
+  const balanceText = ob.balance > 0 ?
```

Comment:
I'm checking > 0, because the balance gets a different style in that case.

As far as the number check earlier, it's just defensive programming so the view doesn't just bomb with unexpected data.